Security page clean up (#504)
ethereum-optimism/community-hub: src/docs/security-model/README.md

lang: en-US
---

> Security in crypto is vital. Users, developers and infrastructure providers all need to trust that the code works, as expected. When it breaks, these same groups (rightfully) demand a fast and secure fix that is clearly communicated. 

Reviewer: Some nitty suggestions:

```suggestion
Security in crypto is vital. Users, developers and infrastructure providers expect the code to work as intended. When it doesn't, these same groups (rightfully) demand a fast, secure, and clearly communicated fix. 
```

Rationale: 
1. I wanted to remove the word 'trust', as I find it quite confusing in this context. Y'know, like does it meet you "trust the code", or "trust us to write secure code". Easier to just work around it. 
2. Expected/intended are similar... but my thought is that most users expectations are vaguely defined. I feel like saying 'intended' is more suggestive of the developers intent.
3. Reordered the last sentence just a bit for flow.

> Over the years at Optimism we have learnt a great deal about this delicate balance, and the importance of transparency around security models. It is not good enough to just be transparent, it also needs to be accessible. If the average user cannot understand your security model then it does not matter how transparent it is. 

Reviewer: Nit: 
~~learnt~~ learned
I think we prefer American english

> In line with this philosophy of radical transparency we have a very accessible run down of [Optimism's security model](./optimism-security-model.md) that anyone can understand. 

Reviewer: ```suggestion
In line with this philosophy of transparency we have a very accessible run down of [Optimism's security model](./optimism-security-model.md) that anyone can understand. 
```

The term "radical" wasn't used in the previous paragraph, and I think it has a very strong (but not clear to me) connotation about transparency that I'd prefer we not necessarily be held to.

Author: I could link to the radical transparency ideology, but I don't think we follow it too closely so we can 100% remove it.
